### PR TITLE
Add legal_entity edrpou check on merge_legal_entities job

### PR DESCRIPTION
**Pull request by progsmile**
#3366

### CODE REVIEW

File: apps/core/test/support/prm_factories/party_factory.ex [hunk 1]
```diff
@@ -12,7 +12,7 @@ defmodule Core.PRMFactories.PartyFactory do
           second_name: "Миколайович",
           birth_date: ~D[1991-08-19],
           gender: "MALE",
-          tax_id: sequence("222222222"),
+          tax_id: random_tax_id(),
```
Comment (PavelVesnin) [inline apps/core/test/support/prm_factories/party_factory.ex]: why not sequence? Random can collide, when tax_id must be unique

File: apps/graphql/test/web/resolvers/legal_entity_merge_job_resolver_test.exs [hunk 2]
```diff
@@ -90,6 +91,26 @@ defmodule GraphQLWeb.LegalEntityMergeJobResolverTest do
...
+
+      resp =
+        conn
+        |> put_client_id(UUID.generate())
+        |> post_query(@query, input_signed_content(signed_content))
+        |> json_response(200)
+
+      resp_entity = get_in(resp, ~w(data legalEntityMergeJobs))
+
+      refute resp_entity
```
Comment (PavelVesnin) [inline apps/graphql/test/web/resolvers/legal_entity_merge_job_resolver_test.exs]: could be in one line

File: apps/core/test/support/prm_factories/party_factory.ex (hunk 1 above)
Comment (progsmile) [reply apps/core/test/support/prm_factories/party_factory.ex]: had issues with unique_constraint

Comment (progsmile) [reply apps/core/test/support/prm_factories/party_factory.ex]: I've check it again

Comment (progsmile) [reply apps/core/test/support/prm_factories/party_factory.ex]: done with sequence under the hood

**Pull request merged**